src/styles/blocks/calendar.scss

```
@@ -7,25 +7,17 @@
   padding: 40px $calendar-side-paddings 0;
 
   &__day {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    position: relative;
     @include square($day-size);
     border: 1px solid #000;
     background-color: #eee;
-    transition: ease 0.5s;
+    transition: background-color, transform 0.5s;
```

This way you set transition time only for transform property, the background color changes immediately
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```